components/Map/index.js

```diff
@@ -13,7 +18,7 @@ import IconLabel from "../IconLabel";
 import { FaExpand } from "react-icons/fa";
 import "mapbox-gl/dist/mapbox-gl.css";
 
-export default function Map({
+export default function MapComponent({
```

Comment:
This was a variable name collision because we get `Map` now out of the `react-map-gl/mapbox` library.